weed/server/filer_server_handlers_tagging.go

```diff
@@ -81,8 +81,10 @@ func (fs *FilerServer) DeleteTaggingHandler(w http.ResponseWriter, r *http.Reque
 	// parse out tags to be deleted
 	toDelete := strings.Split(r.URL.Query().Get("tagging"), ",")
 	deletions := make(map[string]struct{})
-	for _, deletion := range toDelete {
-		deletions[deletion] = struct{}{}
```

Comment:
You can also just fix by checking `deletion == ""` or not.

Reply:
I've updated it accordingly.